parquet-column/src/test/java/org/apache/parquet/column/columnindex/TestColumnIndexBuilder.java

```diff
@@ -227,17 +266,45 @@ public void testStaticBuildBinary() {
         BoundaryOrder.ASCENDING,
         asList(true, true, false, false, true, false, true, false),
         asList(1l, 2l, 3l, 4l, 5l, 6l, 7l, 8l),
-        toBBList(null, null, stringBinary("Beeblebrox"), stringBinary("Dent"), null, stringBinary("Jeltz"), null,
+        toBBList(
+            null,
+            null,
+            stringBinary("Beeblebrox"),
+            stringBinary("Dent"),
+            null,
+            stringBinary("Jeltz"),
+            null,
             stringBinary("Slartibartfast")),
-        toBBList(null, null, stringBinary("Dent"), stringBinary("Dent"), null, stringBinary("Perfect"), null,
+        toBBList(
+            null,
+            null,
+            stringBinary("Dent"),
+            stringBinary("Dent"),
+            null,
+            stringBinary("Perfect"),
+            null,
             stringBinary("Slartibartfast")));
     assertEquals(BoundaryOrder.ASCENDING, columnIndex.getBoundaryOrder());
     assertCorrectNullCounts(columnIndex, 1, 2, 3, 4, 5, 6, 7, 8);
```

Comment:
The check is passed? why not "assertCorrectNullCounts(columnIndex, 11, 21, 31, 41, 51, 61, 71, 81)"

Reply:
When the `List<Long>` is created by using `Arrays.asList` I have to use long values otherwise a `List<Integer>` would be created. `1l`, `2l` etc. are the `long` literals in java. Others might use capital L as such `1L`, `2L` etc.

Reply:
ohhh, I got it wrong. It's letter "l", not number "1".